docs-source/content/samples/simple/domains/domain-home-in-image/_index.md

```diff
@@ -69,7 +69,7 @@ The script will perform the following steps:
 * Create a tag that refers to the generated Docker image.
 * Create a Kubernetes domain YAML file, `domain.yaml`, in the directory that is created above. This YAML file can be used to create the Kubernetes resource using the `kubectl create -f` or `kubectl apply -f` command.
 ```
-$ kubectl apply -f /path/to/output-directory/weblogic-domains/<domainUID>/domain.yaml
+$ kubectl apply -f /<path to output-directory>/weblogic-domains/<domainUID>/domain.yaml
 ```
 
 As a convenience, using the `-e` option, the script can optionally create the domain object, which in turn results in the creation of the corresponding WebLogic Server pods and services. This option should be used in a single node Kubernetes cluster only.
```

Comment:
This seems to be giving options: use `-e` or not. I would suggest that Quick Start should always use it, and that the text should assume it and not give options; however, it should probably point out that it is needed.

Reply:
This is not the Quick Start document. I edited some of the others to clarify the /path/to/output-directory/ problem you pointed out to me.

Reply:
The Quick Start does give the -e option.